docs/the-git-based-process-you-need-for-mainframe-development-DRAFT/the-git-based-development-process-for-mainframe-development.md

```diff
@@ -35,14 +39,49 @@ As Git became the de-facto version control system in today's IT world, new termi
+The `main` branch is the point of reference for the entire history of the mainline changes to the code base. `main` should be a *protected* 
+branch - **all** changes originate on a branch created to hold them until they are ready to be merged. Using a branch-and-merge approach is
+natural in a git-based SCM and very light-weight both in terms of resource consumption and developer experience.
+
+#### All changes start on a dedicated branch
+
+Although all git-based Devops services provide direct access to the server copy of the repo, developers will typically use an IDE of their
+choice to work locally on a *clone* of the repo. Standard practices ensure developers' local clones are synchronized with the server copy
+(typically known as the *remote*) through actions known as `pull` and `push`.
+
+In a small team where there is almost never more than one change in progress at a time, using a branch enables the developer to make
+the changes in a series of commits rather than as one atomic change as they edit source and save files. Of course, the 'one-line' change
+might mean there genuinely is only one commit that needs merging, but the process is so natural and light-weight that it's not 
+worth making that a special case.
+
+Such branches are able to be built prior to merging - this can eliminate the possibility of the merge breaking the build of `main`
+so reducing risk in making changes.
+#### Merging a branch
+
+A branch holds all the commits for a change - be that a single commit for a one-liner or a sequences of commits as the developer refined
+the changes making them ready for review and merging into `main`.
+
+The request to merge a branch is made explicitly, but can be as formal or informal as needed by the team. Protection of `main` can mean that
+only certain people can perform the merge, or that a review of the changes has approved them, or both.
+
+The action of merging can either simply take all the commits from the branch and add them to `main` or the multiple commits in the
+branch can be *squashed* into one commit - the latter can keep the overall history on `main` 'cleaner' if that's important to the 
+team.
+
+`main` will always build - and the team can choose when to
+package and deploy.
+### Scaling-up
+
+The use of branches for concurrently planned activities scales extremely well for busier teams. Additionally, *epic*[^4] and *release maintenance* branches accommodate specific development workflows and allow the model to scale even further.
```

Comment:
> The use of branches for concurrently planned activities scales extremely well for busier teams

Maybe introduce "integration branches"?

The use of integration branches helps to scale in busier teams for coordinating concurrently planned activities and scales very well.